pkg/serving/config_changes.go

```diff
@@ -235,17 +242,17 @@ func UpdateLabels(service *servingv1alpha1.Service, template *servingv1alpha1.Re
 
 // =======================================================================================
 
-func updateEnvVarsFromMap(env []corev1.EnvVar, vars map[string]string) []corev1.EnvVar {
+func updateEnvVars(env []corev1.EnvVar, toUpdate map[string]string) []corev1.EnvVar {
```

Comment:
I'd say we revert this change, `updateEnvVarsFromMap` is explicit. Also we have exported method 'UpdateEnvVars' (same name), IMO we should avoid using same function names.